specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/nova.json

```diff
@@ -0,0 +1,1051 @@
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ClusterResource"
+            },
+            "description": "Parameters to provide for specifying the managed Cassandra cluster."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The managed Cassandra cluster is being updated. Poll for provisioningState=Succeeded to find when creation completes.",
+            "schema": {
+              "$ref": "#/definitions/ClusterResource"
+            }
+          },
+          "201": {
+            "description": "The managed Cassandra cluster is being created. Poll for provisioningState=Succeeded to find when creation completes.",
+            "schema": {
+              "$ref": "#/definitions/ClusterResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/DefaultErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "CassandraCluster_Update",
+        "description": "Updates a managed Cassandra cluster.",
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "CosmosDBManagedCassandraClusterPatch": {
+            "$ref": "./examples/CosmosDBManagedCassandraClusterPatch.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ClusterResource"
+            },
+            "description": "Parameters to provide for specifying the managed Cassandra cluster."
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The managed Cassandra cluster is being updated. Poll for provisioningState=Succeeded to find when the update completes.",
+            "schema": {
+              "$ref": "#/definitions/ClusterResource"
+            }
+          },
+          "200": {
+            "description": "The managed Cassandra cluster has been updated.",
+            "schema": {
+              "$ref": "#/definitions/ClusterResource"
+            }
+          },
```

Comment:
I was thinking, can we make status codes for PUT and PATCH consistent. Both of them are individually correct, but why are they not same ? Any particular reason ?

Reply:
Oversight. I've made them consistent.